### DISCUSSION TITLE
Blackhole-> Firefox

### DISCUSSION THREAD

**Discussion by timboxyz** [ExistentialAudio/BlackHole#94] [General]
Firefox (75.0 just tested and the previous version too) on 10.14.6 doesn't seem to recognise Blackhole as a valid audio source. Chrome, however, is fine for example

**Comment by devinroth** [comment]
Strange. I’ll take a closer look.

**Comment by timboxyz** [comment]
Even stranger, this morning FF now shows blackhole as the *only* audio source and won't let me select the built-in Mic!

I am suspecting this is more a FF issue.

**Comment by timboxyz** [comment]
> Even stranger, this morning FF now shows blackhole as the *only* audio source and won't let me select the built-in Mic!
> 
> I am suspecting this is more a FF issue.

In fact, probing further, it only seems to recognise the currently selected default in sound preferences. The above change was due to that 'feature', so an OSX<->FF issue I'm sure now. But one to be aware of!

**Comment by devinroth** [comment]
Thanks for figuring it out. I’ll leave this up in case it helps someone out.

**Comment by DJAmos** [comment]
Morning all 

I cant seem to playout to FB  on firefox version 75.0 (64-bit)Version 81.0.4044.129 (Official Build) (64-bit) how ever on Chrome it play interminently then i loose sound.

Also as per your tutotrial when you select Blackhole as a default output it chooses only speaker, when i select this it shows the speaker icon and the microphone speaker.

is this is a known problem or is there a fix ?

PS I am using a MacBook ...

Regards 

Amos

**Comment by disambiguator** [comment]
Sorry if this is a spam, but I also had this issue and switching to Chrome worked. If I didn't see this issue, I would have been scratching my head for much longer. Thank you!

**Comment by eofreternal** [comment]
2021, still an issue
